### PR TITLE
Refactor tle handling

### PR DESCRIPTION
**Pull request by adybbroe**
Re factor the TLE handling a bit and add option to fetch TLEs from an other local repository/directory (NFS or similar) and copy them into the AAPP data structure.

 - [ ] Closes #xxxx 
 - [x] Tests added 
 - [x] Tests passed: Passes ``pytest`` 
 - [ ] Passes ``flake8`` 
 - [ ] Fully documented

### CODE REVIEW

**Comment by adybbroe** [comment]
Not sure I can devote much more time at this. Not easy to make meaningful unittests to the TLE file handling!

**Comment by mraspaud** [comment]
Do you want us to review it now or do you plan on improving this later?

**Comment by adybbroe** [comment]
Ok, if I merge and close this one? 
I would like to try make a release of how it is now and then we can make new attempts to refactor and improve test coverage after that?
@mraspaud @gerritholl ?

**Comment by mraspaud** [comment]
I haven't reviewed it yet

File: aapp_runner/tle_satpos_prepare.py [hunk 1]
```diff
@@ -200,24 +231,31 @@ def do_tleing(config, timestamp, satellite):
 
     _ensure_tledir(DIR_DATA_TLE)
 
+    # Fetch TLE files from central real-time repo and place them under the AAPP orbelems structure:
+    if 'recent_tlefiles_ext_dir' in config['aapp_processes'][config.process_name]:
```
Comment (mraspaud) [inline aapp_runner/tle_satpos_prepare.py:235]: This new config isn't documented anywhere. Please add it to the documentation and config examples.

File: aapp_runner/tle_satpos_prepare.py [hunk 2]
```diff
@@ -178,6 +182,33 @@ def download_tle(config, timestamp, dir_data_tle):
     return tle_file_list
 
 
+def fetch_realtime_tles(tle_input_path, tle_output_path, tle_infile_format):
```
Comment (mraspaud) [inline aapp_runner/tle_satpos_prepare.py:187]: This function needs to be tested.

**Review by mraspaud — CHANGES_REQUESTED** [review]
Thanks for adding this feature. Please document and test it.

File: aapp_runner/tle_satpos_prepare.py (hunk 2 above)
Comment (adybbroe) [reply aapp_runner/tle_satpos_prepare.py:187]: I have added a test now. Let me hear what you think?

File: aapp_runner/tle_satpos_prepare.py (hunk 1 above)
Comment (adybbroe) [reply aapp_runner/tle_satpos_prepare.py:235]: I have added an explanation in the template config file.

**Review by mraspaud — APPROVED** [review]
LGTM! Thanks so much for finishing this!

**Comment by adybbroe** [comment]
Many thanks for reviewing @mraspaud ! :-)

**Pull request merged**